### ISSUE TITLE
[FEAT] Symlink, rather than copy, php-local.ini

### ISSUE THREAD

**Issue by davidnewhall** [linuxserver/docker-baseimage-alpine-nginx#121]
### Is this a new feature request?

- [X] I have searched the existing issues

### Wanted change

Currently the container init script copies a few files from `/config` into their correct locations within `/etc`. I am requesting these be symlinks instead of copies. The specific file I'm calling out is `php-local.ini`, but there are probably others that can benefit from a symlink.

### Reason for change

I edited `/config/php/php-local.ini` and expected that change to propagate to php-cli immediately, but it did not. My options for making it work are 1) Restart the container, or 2) Shell into the container and edit the correct file in `/etc/`.

Turning the files in /etc into symlinks would make changes to php-local.ini immediately usable by php-cli; without restarting the container and without shelling into it.

I run several php apps out of the swag container, and they all have code that may exec `php` as a cli. The websites also receive a lot of traffic, so restarting the container to update a non-running config is out of the question. I was able to work around the current setup by opening a shell on the container and editing the file directly in `/etc/php8/`.

### Proposed code change

tl;dr `s/cp/ln -s`
